daemon/src/connection.rs

```diff
@@ -364,19 +364,19 @@ impl Actor {
                 )
             })?
             .with_context(|| format!("Failed to read first message from maker {maker_identity}"))? {
-            Some(wire::MakerToTaker::Hello(maker_version)) => {
-                if our_version != maker_version {
+            Some(wire::MakerToTaker::Hello(actual_version)) => {
```

Comment:
❓ Is it a little weird that we get the same regular `Hello` message from the maker whether they agree with our version or not?

Reply:
It is not about agreeing, it is a statement / confirmation of "we will be using this version".